oncotator/output/RecordBuilder.py

```diff
@@ -289,6 +289,11 @@ def addInfo(self, sampleName, ID, num=None, dataType="String", val=None, isSplit
             self._determineVal2FixedNumField(self._info, ID, nalts, isSplit, val)
         elif num == 0:  # num is either true or false
             if ID not in self._info:
+
```

Comment:
Yes I'm sure.  The new 1000g ds was already preventing a unitest from passing so it didn't seem necessary to write a new one.  From now on, we should rerun all tests using the new db-dir before releasing a new corpus...